test/scan.test.ts

```diff
@@ -1301,11 +1301,11 @@ describe('scan', () => {
     try {
       await scan({
         path: fixturePath,
-        '--max-depth': -1,
+        'max-depth': -1,
       });
     } catch (err) {
       expect(err.message).toEqual(
-        'Could not scan C/C++ project: invalid options: --max-depth should be a positive number.',
+        'Could not scan C/C++ project: invalid options: max-depth should be a positive number.',
```

Comment:
hmm if the user makes use of --max-depth, and if this error-message is printed to the user, then I guess it should include the dashes.